### PR TITLE
Update extauthz example

### PR DESCRIPTION
**Pull request by johnzheng1975**
**Please provide a description of this PR:**
Previous there is a defect, namespace is missing, the sample cannot works successfully
Fix this typo.

### CODE REVIEW

File: samples/extauthz/README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Note that `a` is just a default value for testing. It can be changed with the fl
     Send a check request with header `x-ext-authz: allow` to the Ext Authz server:
 
     ```console
-    $ kubectl exec -it $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -c sleep -- curl -v ext-authz:8000 -H "x-ext-authz: allow"
+    $ kubectl exec -it $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -n foo -c sleep -- curl -v ext-authz:8000 -H "x-ext-authz: allow"
```
Comment (kfaseela) [inline samples/extauthz/README.md:33]: At line 27, I do not see sleep pod being deployed in foo ns

Comment (johnzheng1975) [reply samples/extauthz/README.md:33]: In https://istio.io/latest/docs/tasks/security/authorization/authz-custom/, it should use foo.

Comment (johnzheng1975) [reply samples/extauthz/README.md:33]: That's why it show foo here.
image

Two way to fix it.
1. add foo namespace everywhere.
2. remove foo namespace everywhere.

To make consistent with other doc, can I suggest to go to way 1. Thanks.

**Comment by johnzheng1975** [comment]
@kfaseela 
I agree your comment:
> At line 27, I do not see sleep pod being deployed in foo n

However, this docs do show `-n foo` in its document
https://github.com/istio/istio/blob/master/samples/extauthz/README.md?plain=1#L33
image

So, there is a error, user cannot run successful, right?
Thanks for checking again, and review again, thanks!

**Comment by ericvn** [comment]
Have you tried using the sample as documented without using the `foo` namespace? I suspect that it would work fine. There really isn't a need for the `foo` namespace, The docs use the `foo` space just to keep some things cleaner.

So `foo` isn't needed. If you want to add it, you need to add it for all the relative commands. You also need to add instructions for creating and deleting the namespace (which you don't have).

**Comment by johnzheng1975** [comment]
Yes, it can work. (I can do a test tomorrow)
However, do not you think it is inconsistent with other docs, like: https://istio.io/v1.15/docs/tasks/security/authorization/authz-custom/
which also used foo.    
Just FYI  :-)

**Comment by johnzheng1975** [comment]
@ericvn  @kfaseela 
Thanks for your suggestion. I use **way 2 (remove foo namespace everywhere)** to fix it:
Not I remove foo namespace. Test Pass.
Please review again.

**Review by ericvn — APPROVED** [review]

**Pull request merged**